src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -701,20 +725,39 @@ public void SetRigidity(int rigidity)
 
         var action = new MicrobeEditorAction(this, cost,
             DoRigidityChangeAction,
-            UndoRigidityChangeAction,
+            UndoRigidityChangeAction, nameof(SetRigidity),
             new RigidityChangeActionData(newRigidity, prevRigidity));
 
         EnqueueAction(action);
     }
 
     /// <summary>
-    ///   Removes organelles under the cursor
+    ///   Show options for the organelle under the cursor
     /// </summary>
     [RunOnKeyDown("e_secondary")]
-    public void RemoveOrganelle()
+    public void ShowOrganelleOptions()
     {
         GetMouseHex(out int q, out int r);
 
+        var organelle = editedMicrobeOrganelles.GetOrganelleAt(new Hex(q, r));
+
+        if (organelle == null)
+            return;
+
+        gui.ShowOrganelleMenu(organelle);
```

Comment:
Or perhaps the check against MovingOrganelle being null could be here instead of the GUI, as I think that makes more sense.